.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
+.rspec
```

Comment:
Vot šoreiz šo nevajag komentet.
Tas ir papild atributi ar kuriem tiek izsaukts rspec komanda no konsoles

Reply:
šis varētu būt diskutabls, jo no otrs puses tie varētu būt personalizētie uzstādījumi - vienam patīk laist ar krāsām rspec, citam piemēram terminālī nav krāsu supporta (kkadam linux gīkam).

Bet in general mēs izvēlamies šo komitot, bet tas katrā ziņā nav make or break  gadījums.